packages/ifpack2/src/Ifpack2_ILUT_def.hpp

```diff
@@ -499,8 +492,10 @@ void ILUT<MatrixType>::compute ()
 
     Array<scalar_type> InvDiagU (myNumRows, zero);
 
-    Array<local_ordinal_type> tmp_idx;
-    Array<scalar_type> tmpv;
+    Array<Array<local_ordinal_type> > L_tmp_idx(myNumRows);
```

Comment:
Given that ILUT already fills the matrix by rows, it seems like the next logical step would be to get rid of this intermediate data structure and do a count-allocate-fill thing.  Of course it's good to know whether these changes can make it through all the tests first.

Reply:
See comments below.